src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
random should be class-level `private final` field, its mentioned in checklist

Reply:
not fixed

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/156338484/fc1681df-41ff-4557-91df-27fb5620b890)
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md